webclient/src/views/view/view-view.inc

```diff
@@ -181,6 +181,12 @@
         >
           ${{_.view_view.msg.no_floor_overlay}}$
         </div>
+        <div
+          class="toast"
```

Comment:
I would think, that for the cases we need, a `toast-warning` would be better. 
What do you think design-wise?

Reply:
My idea was that comments can be warnings, but also just informative. That is why I would prefer a neutral toast.